docs/adr/0013-sdk-crypto-domain.md

```
@@ -0,0 +1,133 @@
+
+```sh
+cd pkg/v1 && GOWORK=off go list -deps ./crypto/... | grep -i 'x/crypto'  # empty
+```
+
+Future schemes (XChaCha20-Poly1305, with its 192-bit nonce for high-volume
+random-nonce workloads) land under `third-party/x-crypto/*` and activate only
+via their own explicit blank import — never auto-pulled into `pkg/v1`.
+
+## Error-code allocation (amends ADR 0005)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)